addons/spreadsheet/static/tests/hoot_override.js

```diff
@@ -0,0 +1,16 @@
+import { getRunner } from "@web/../lib/hoot/main_runner";
+import { patch } from "@web/core/utils/patch";
+import { stores } from "@odoo/o-spreadsheet";
+
+const { RendererStore } = stores;
+
+patch(RendererStore.prototype, {
+    drawLayer(ctx, layer) {
+        const runner = getRunner();
+        if (runner.debug) {
+            return super.drawLayer(ctx, layer);
+        } else {
```

Comment:
I'd add a comment saying what is the purpose of this patch

Reply:
dones :)